### PR TITLE
Product layout card finish

### PR DESCRIPTION
**Pull request by WaveOfDandelions**
- [DEMO LINK](https://WaveOfDandelions.github.io/layout_product-cards/)
- [TEST REPORT LINK](https://WaveOfDandelions.github.io/layout_product-cards/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -4,10 +4,56 @@
...
+            </div>
+          </div>
+
+          <div class="item__price-container price">
+            <p class="price__text">Price:</p>
+            <p class="price__value">$2,199</p>
+          </div>
+
+          <button class="item__button" type="submit">
+            <a class="item__link" href="/productLink"
```
Comment (Oskinavara) [inline src/index.html:49]: Formatting is really weird here. Please use the autoformatter

File: src/styles/blocks/_card.scss [hunk 2]
```diff
@@ -0,0 +1,78 @@
+.card {
+  width: 198px;
+  @include border(1px, solid, $border-color, 5px);
+  .card__item {
```
Comment (Oskinavara) [inline src/styles/blocks/_card.scss:4]: Such nesting is unnecessary and increases the specificity of the selector which is generally bad. Use SCSS nesting instead.

File: src/styles/blocks/_card.scss [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+      margin-bottom: 16px;
+      .price__text {
+        @include font(400, 12px, 18px, $font-color--secondary, Roboto);
+      }
+      .price__value {
+        @include font(700, 16px, 18px, $font-color, Roboto);
+      }
+    }
+    .item__button {
+      height: 40px;
```
Comment (Oskinavara) [inline src/styles/blocks/_card.scss:55]: Setting fixed widths and heights is generally a bad idea. Width of the element can be determined by it's parent and set to 100%. Height could be determined by line-height + paddings.

File: src/styles/blocks/_card.scss [hunk 4]
```diff
@@ -0,0 +1,78 @@
+.card {
+  width: 198px;
```
Comment (Oskinavara) [inline src/styles/blocks/_card.scss:2]: It would be better to have max-width: 198px and width: 100%; so it can be narrower if needed

**Review by Oskinavara — CHANGES_REQUESTED** [review]

File: src/index.html (hunk 1 above)
Comment (WaveOfDandelions) [reply src/index.html:49]: > Formatting is really weird here. Please use the autoformatter

I use htmlint and editor config with 2 spaces, can you explaine me how it should be?

File: src/styles/blocks/_card.scss (hunk 2 above)
Comment (WaveOfDandelions) [reply src/styles/blocks/_card.scss:4]: > Such nesting is unnecessary and increases the specificity of the selector which is generally bad. Use SCSS nesting instead.

If I nest like this .card__item inside .card it is not scss nesting??

**Review by choeqq — CHANGES_REQUESTED** [review]
Waiting for any updates @WaveOfDandelions 👀

File: src/index.html (hunk 1 above)
Comment (WaveOfDandelions) [reply src/index.html:49]: > Formatting is really weird here. Please use the autoformatter

Fixed

File: src/styles/blocks/_card.scss (hunk 2 above)
Comment (WaveOfDandelions) [reply src/styles/blocks/_card.scss]: > Such nesting is unnecessary and increases the specificity of the selector which is generally bad. Use SCSS nesting instead.

Done

File: src/styles/blocks/_card.scss (hunk 3 above)
Comment (WaveOfDandelions) [reply src/styles/blocks/_card.scss]: > Setting fixed widths and heights is generally a bad idea. Width of the element can be determined by it's parent and set to 100%. Height could be determined by line-height + paddings.

I was change properties for better flexibility

File: src/styles/blocks/_card.scss (hunk 4 above)
Comment (WaveOfDandelions) [reply src/styles/blocks/_card.scss]: > It would be better to have max-width: 198px and width: 100%; so it can be narrower if needed

Done

**Review by choeqq — APPROVED** [review]
Looks good ✅

**Pull request closed without merging**